### ISSUE TITLE
[Accessibility: ACC-255.1] Icon: No text alternative for non-text UI.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12588]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
For all non-text UI elements text alternatives should be provided and this information should be read out by screen reader.
Every semantic piece of non-text content should be available as text.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Platform table](https://sap.github.io/fundamental-ngx/#/platform/table/basic)
There is no text alternative provided to ‘Search’ and ‘Synchronize’ icons within all the ‘Search’ fields.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/1e2fa338-a9f5-44bd-82e8-965c7697bdaf">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 I tested our platform table with a toolbar via JAWS and everything is announced properly. See the screenshot. If you have issues, please, report a new issue with stackblitz example and steps how to reproduce the issue.
<img width="1145" alt="Screenshot 2024-10-21 at 14 35 45" src="https://github.com/user-attachments/assets/d192e0fb-8c6c-468b-921f-6438c4b1466b">
